src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Cohort/Review.cshtml

```diff
@@ -9,8 +9,7 @@
 }
 
 <div class="govuk-grid-row">
-    <div class="govuk-grid-row">
-        <div class="govuk-width-container">
+        <div class="govuk-grid-column-full">
```

Comment:
Was this change deliberate?

Reply:
James spotted that in a view yesterday and he fixed it, the html wasn't right.